consumer_test.go

```diff
@@ -1342,3 +1343,108 @@ func Test_partitionConsumer_parseResponse(t *testing.T) {
 		})
 	}
 }
+
+func (b *appendInterceptor) onConsume(msg *ConsumerMessage) {
+	msg.Value = []byte(string(msg.Value) + strconv.Itoa(b.i))
+	b.i++
+}
```

Comment:
Should these all be defined together somewhere? It seems weird that over in this file we're adding methods to the type we assume is defined in another file?

Maybe that's a normal Go thing, though. If so, carry on 😄

Reply:
not necessarily, I had the same issue when navigating go source, then I read that the tooling can give you all the methods that a type implements, and you shouldn't rely only in what's defined in a file.
but if that helps reading the code, I'll move it.

this is what the tooling currently shows:
```
/Users/diegoalvarez/src/github.com/Shopify/sarama/async_producer_test.go:1234.6-1234.22: definition of type appendInterceptor (size 8, align 8)
/Users/diegoalvarez/src/github.com/Shopify/sarama/async_producer_test.go:1234.6-1234.22: Methods:
/Users/diegoalvarez/src/github.com/Shopify/sarama/consumer_test.go:1347.29-1347.37: 	method (*appendInterceptor) onConsume(msg *ConsumerMessage)
/Users/diegoalvarez/src/github.com/Shopify/sarama/async_producer_test.go:1238.29-1238.34: 	method (*appendInterceptor) onSend(msg *ProducerMessage)
/Users/diegoalvarez/src/github.com/Shopify/sarama/async_producer_test.go:1234.6-1234.22: Fields:
/Users/diegoalvarez/src/github.com/Shopify/sarama/async_producer_test.go:1235.2-1235.2: 	i int
```